src/uct/api/uct.h

```diff
@@ -1928,10 +1932,8 @@ UCT_INLINE_API ucs_status_t uct_ep_tag_rdnv_cancel(uct_ep_h ep, void *op)
  * @ingroup UCT_TAG
  * @brief Send software rendezvous request.
  *
- * Send a rendezvous request only, which will allow the peer to complete the
- * data transfer in software. If the peer has posted corresponding receive it
- * will be reported as incomplete match with both data address and key equal to
- * NULL. Otherwise it will be reported as unexpected.
+ * This routine sends a rendezvous request only, which will allow the peer to
```

Comment:
"This routine sends a rendezvous request only, which will allow the peer" - does not read well.